addon/helpers/is-empty.js

```diff
@@ -0,0 +1,6 @@
+import { helper } from '@ember/component/helper';
+import { isEmpty } from '@ember/utils';
+
+export default helper(function([obj]) {
```

Comment:
Could it just be `export default helper(([obj]) => isEmpty(obj));`?

Reply:
It sure could. Or even
```js
export default helper(compose(first, isEmpty))
```
if you want to go super-functional-programming. I tried to stay stylistically similar to the existing code in this repo.